OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -204,6 +206,11 @@ public bool IsCellAcceptable(Actor self, CPos cell)
 			return Info.TerrainTypes.Contains(terrainType);
 		}
 
+		bool HasAmmo()
```

Comment:
For completion we could do

```suggestion
		bool HasAmmo => ammoPools.Any(p => p.HasAmmo);
```

Though really it's used only in one place so we don't really need to have this abstracted here

Reply:
Yea think it can be removed actually, only added it originally because it made the line where it's called a bit long, but now since it's only checking HasAmmo it'd be fine.